doc/nrf/index.rst

```diff
@@ -3,6 +3,17 @@
 Welcome to the |NCS|!
 #####################
 
+.. important::
+
+   |NCS| v2.1.99-dev1 is a development tag and will be replaced by v2.2.0 in the future.
+
+   The development tag contains the following major updates, marked as experimental:
+
+   * Added ANT protocol support to |NCS|.
```

Comment:
Since there is only one bullet point, may be this could be another option:
```suggestion
   The development tag contains the addition of ANT protocol to the |NCS| as a major update, and the feature is marked as experimental.
```
In case we are going with the existing version,
```suggestion
   The development tag contains the following major update, marked as experimental:

   * Addition of ANT protocol support to the |NCS|.
```
```